### DISCUSSION TITLE
Sessions in DB

### DISCUSSION THREAD

**Discussion by KellerKev** [emmett-framework/emmett#338] [Q&A] [answered]
I see I can store sessions in Redis, filesystem and cookies. Is there also a way to store them in the DB?

Thanks again!

**Comment by gi0baro** [comment, marked as answer]
@Kkeller83 No, there's no provided facility to store sessions in DB.

This is actually a _design decision_, as in my experience sessions cause too much load on relational databases, and in general I strongly suggest not to do that.

It would be theoretically still be possible writing a custom session Pipe for that, subclassing `BackendStoredSessionPipe ` (https://github.com/emmett-framework/emmett/blob/master/emmett/sessions.py#L130)

**Reply by KellerKev** [reply]
Thanks! That is fine, I am happy with redis. Was just curious.
